src/functions/edi/inbound/__tests__/handler.success-with-ack.ts

```diff
@@ -136,6 +150,13 @@ for (const uploadDirectory of ["inbound", "processed"]) {
         "delivered guide JSON to destination webhook"
       );
 
+      const replicationPutArgs = buckets.commandCalls(PutObjectCommand, {
+        bucketName: "core-ingestion-bucket",
+      })[0]!.args[0].input;
+
+      t.assert(replicationPutArgs.key === "replicated-documents/inbound.edi");
```

Comment:
What do you think about adding a suffix with a timestamp? Thinking of our larger customers, they have incoming files re-using the same name. There could be file overwrites if we re-use the incoming name on its own.